eng/pipelines/templates/variables/globals.yml

```diff
@@ -1,3 +1,3 @@
 variables:
   skipComponentGovernanceDetection: true
-  DotNetCoreVersion: '2.1.503'
\ No newline at end of file
+  DotNetCoreVersion: '3.1.405'
```

Comment:
@mikeharder any particular reason you picked this version of .NET Core? It seemed to have broken our pipeline notification pipeline (https://dev.azure.com/azure-sdk/internal/_build/results?buildId=696294&view=logs&j=f2ee0415-d5cb-5e2e-67d5-dd79ca5661b0&t=7464715b-8d60-5b1f-42f6-640a0d60e970) We might need to update that project to use a newer .NET Core as well.

Reply:
@danieljurek has a PR https://github.com/Azure/azure-sdk-tools/pull/1339 out to address this.